config/monitoring/metrics/prometheus/100-grafana-dash-knative-scaling.yaml

```diff
@@ -563,6 +563,30 @@ data:
                   "intervalFactor": 1,
                   "legendFormat": "Excess Burst Capacity",
                   "refId": "D"
+               },
+               {
+                  "expr": "sum(autoscaler_stable_requests_per_second{namespace_name=\"$namespace\", configuration_name=\"$configuration\", revision_name=\"$revision\"})",
+                  "format": "time_series",
+                  "interval": "1s",
+                  "intervalFactor": 1,
+                  "legendFormat": "Average RPS",
+                  "refId": "E"
+               },
+               {
+                  "expr": "sum(autoscaler_panic_requests_per_second{namespace_name=\"$namespace\", configuration_name=\"$configuration\", revision_name=\"$revision\"})",
+                  "format": "time_series",
+                  "interval": "1s",
+                  "intervalFactor": 1,
+                  "legendFormat": "Average Panic RPS",
+                  "refId": "F"
+               },
+               {
+                  "expr": "sum(autoscaler_target_requests_per_second{namespace_name=\"$namespace\", configuration_name=\"$configuration\", revision_name=\"$revision\"})",
+                  "format": "time_series",
+                  "interval": "1s",
+                  "intervalFactor": 1,
```

Comment:
I think we need to change the `title` for this dashboard as well or create a new one for RPS.